package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@emotion/core": "^10.0.22",
     "@emotion/styled": "^10.0.23",
+    "@material-ui/core": "4.9.8",
```

Comment:
what is the point of using `material-ui `?

Reply:
I used it for Backdrop functionality. I'll remove it right away.